### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by lewinther**
Implemented DB and continued working on pages and logical structures. 
Going forward we will discuss how the app is accessed as well as how to implement chat-logic, a login page, and prettifying the incoming data.

### CODE REVIEW

**Comment by lewinther** [comment]
Can we merge the app?

File: src/components/NewsCard.js [hunk 1]
```diff
@@ -1,13 +1,27 @@
...
 
   useEffect(() => {
     async function fetchData() {
-      const res = await fetch("../data/news.json");
-      const newsData = await res.json();
-      setData(newsData);
+      const News = Parse.Object.extend("News");
+      const query = new Parse.Query(News);
+      query.descending("createdAt"); // Assuming you want the latest news article
+      query.first().then((latestNews) => {
```
Comment (igawaclawska) [inline src/components/NewsCard.js:12]: Sorting news by the most recent is the right decision. :)

File: src/components/ChatListHome.js [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+    // Combined query for either sent or received messages.
+    const combinedQuery = Parse.Query.or(sentMessagesQuery, receivedMessagesQuery);
+    combinedQuery.descending('Message_Date');
+    combinedQuery.select('Sender_User_ID', 'Receiver_User_ID', 'Message_Text', 'Message_Date'); // Make sure to select 'Message_Date' and 'Message_Text'.
+
+    combinedQuery.find().then(results => {
+      // Use a Map to track the most recent message for each chat partner.
+      const chatPartnersMap = new Map();
+
```
Comment (igawaclawska) [inline src/components/ChatListHome.js:33]: Well done! Also, great job on adding comments

File: tests/db_helper.js [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+const user = new User();
+
+// Create dummy user id for testing queries
+user.set("dock", "P1");
+user.set("username", "Mealz");
+user.set("email", "alex.mealzy@cbs.dk");
+user.set("phone_no", "12345678");
+user.set("active_user", true);
+user.set("password", "1234");
+
```
Comment (igawaclawska) [inline tests/db_helper.js:22]: 👍

**Review by igawaclawska — APPROVED** [review]
Looks good! Approved :)

**Pull request closed without merging**